spinnaker/provider.go

```diff
@@ -1,19 +1,21 @@
 package spinnaker
 
 import (
+	"log"
+
+	spin_config "github.com/estebangarcia/spin/config"
+	gate "github.com/estebangarcia/spin/gateclient"
 	"github.com/hashicorp/terraform/helper/schema"
-	"github.com/spf13/pflag"
-	gate "github.com/spinnaker/spin/cmd/gateclient"
 )
 
 func Provider() *schema.Provider {
 	return &schema.Provider{
 		Schema: map[string]*schema.Schema{
 			"server": {
 				Type:        schema.TypeString,
-				Required:    true,
+				Optional:    true,
 				Description: "URL for Gate",
-				DefaultFunc: schema.EnvDefaultFunc("GATE_URL", nil),
+				DefaultFunc: schema.EnvDefaultFunc("SPIN_GATE_ENDPOINT", nil),
```

Comment:
Changing this may break existing users. The original name is definitely not great since it isn't using PROVIDER_ATTR format, but maybe this change should be a separate PR so that it's easier to accept this one.